modules/benchmark/bench_results.c

```diff
@@ -371,7 +390,7 @@ char *bench_result_benchmarkconf_line(bench_result *b) {
     char *bv = bench_value_to_str(b->bvalue);
 
 #define prep_str(s) (s ? (char*)auto_free(gg_key_file_parse_string_as_value(s, '|')) : "")
-    char *ret = g_strdup_printf("%s=%s|%d|%s|%s|%s|%s|%d|%d|%d|%d|%s|%s|%d|%d\n",
+    char *ret = g_strdup_printf("%s=%s|%d|%s|%s|%s|%s|%"PRId64"|%d|%d|%d|%s|%s|%d|%d|%d|%"PRId64"|%s\n",
```

Comment:
This is becoming unwieldy...

Reply:
A little, but it remains compatible with existing results.